hastexo/static/html/main.html

```diff
@@ -28,15 +30,15 @@ <h1 class="header">Sorry!</h1>
   </div>
   <div class="dialog" id="check_pending">
     <div class="inner">
-      <h1><span>Please wait</span><span class="loader"></span></h1>
-      <p class="message">Working</p>
+      <h1><span>{% trans "Please wait" %}</span><span class="loader"></span></h1>
+      <p class="message">{% trans "Working" %}</p>
     </div>
   </div>
   <div class="dialog" id="check_complete">
     <div class="inner">
-      <h1 class="check_result_heading">Progress check result</h1>
-      <p class="check_result_message">You completed <strong class="check_pass">0</strong> out of <strong class="check_total">0</strong> tasks.</p>
-      <h2 class="hints_title">Hints</h2>
+      <h1 class="check_result_heading">{% trans "Progress check result" %}</h1>
+      <p class="check_result_message">{% trans "You completed" %} <strong class="check_pass">0</strong> {% trans "out of" %} <strong class="check_total">0</strong> {% trans "tasks." %}</p>
```

Comment:
I'm still not quite sure if this is the best way to do this. I think you might be better off using [`trans` with placeholders](https://jinja.palletsprojects.com/en/3.0.x/templates/#i18n) here. Would you agree?

Reply:
I agree that this is probably not the best way but we can't use those placeholders here because we are not passing the values in the context when we render the template, we are attaching them in javascript. I tried to build it as one string and get it translated in js but all my attempts failed so far. But, I just might have figured where I went wrong, I'll try again to tweak the js code.

Reply:
I updated the PR now, I defined the string in js, translate with placeholders and then replace the values after translation is loaded. Should be better.